### DISCUSSION TITLE
SBAS Problem    Segmentation fault

### DISCUSSION THREAD

**Discussion by zuaZua160946** [gmtsar/gmtsar#568] [Q&A]
There was a mistake when I carried out SBAS operation. I have tried many times, but it is always the same. How should I solve it?
![Y7CPO VC{43%NHVWDV)CL{9](https://user-images.githubusercontent.com/118714511/202983726-923236c0-8914-4427-84e3-f415dc1bfde8.jpg)

**Comment by Xiaohua-Eric-Xu** [comment]
Do you have enough RAM to do the proposed process? i.e. 64.65059 GB.

If not try reduce your interferogram size or turn on the -mmap option.

**Reply by zuaZua160946** [reply]
Thank you for your reply, the RAM of the server is 128G, the free RAM rarely reaches 64G, the available RAM can exceed 64G, I don't know whether the available RAM meets the requirements, I will try your suggestion, thank you again for your reply.
<img width="368" alt="d3f4f6af77256f39bb9bc0c954e1bac" src="https://user-images.githubusercontent.com/118714511/203056389-86a91928-06e8-4a36-b2fd-2b5c8597cb13.png">

**Reply by Xiaohua-Eric-Xu** [reply]
I am not sure. Could you maybe reduce the size to a safe number (e.g. half) and run it again? Just to make sure it is not from other causes. 

If your disk speed is fast, -mmap could reduce memory usage significantly but will make things run a bit slower. You could however, manually compile sbas_parallel (with a c-compiler that supports openmp) to speed things up by a lot.
